docker/compose/docker-compose.yml

```diff
@@ -66,6 +66,7 @@ services:
     command: controller
     environment:
       WAIT_FOR: zookeeper:2181
+      ZK_URL: zookeeper:2181
       JAVA_OPTS: |
         -Dcontroller.service.port=9090
         -Dcontroller.service.restPort=10080
```

Comment:
Since you are setting the ZK endpoint on `ZK_URL`, you can remove the `-Dcontroller.zk.url=zookeeper:2181` parameter sent on the `JAVA_OPTS`.

Reply:
Sure, thanks. I've pushed a commit with the suggested change.